packages/cypress/src/task/index.ts

```diff
@@ -118,13 +126,17 @@ const customViewConfig = async (
   });
 
   const customViewConfig =
-    allCustomEntityConfigs[CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH];
+    allCustomEntityConfigs[
+      `${CUSTOM_VIEW_HOST_ENTRY_POINT_URI_PATH}-${options.templateName}`
+    ];
```

Comment:
Later, the cached custom view config is retrieved based on the provided `templateName`